text/0000-add-state-field.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Add State to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- State: Approved | On Hold
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add a `State:` field to the `Meta` section of the RFC template.
+
+# Definitions
+[definitions]: #definitions
+
+**Core team:** The party that votes on the approval of RFCs.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC formally introduces the **"On Hold"** state for an RFC, in a way that's clear to the community.
+
+# What it is
+[what-it-is]: #what-it-is
+
+A new `State:` field in the `Meta` section of the RFC template. See [#meta](#meta) for example.
+
+The acceptable values are distinguished in the following ways:
+- **Approved**: The Core team has approved the RFC and is accepting PRs for its implementation.
+- **On Hold**: The Core team has approved the RFC, but there's since been contention about aspects of the proposal. PRs are currently not being accepted for its implementation, until these issues can be resolved.
```

Comment:
```suggestion
- **On Hold**: The Core team has approved the RFC, but PRs are currently not being accepted for its implementation. A reason must be provided in the RFC.
```
I suggest we require individual RFCs to specify why they might be in the On Hold state, and not define a reason for On Hold in this RFC.